src/Service.Tests/SqlTests/GraphQLMutationTests/DwSqlGraphQLMutationTests.cs

```diff
@@ -0,0 +1,604 @@
+        {
+            string msSqlQueryToVerifyDeletion = @"
+                SELECT COUNT(*) AS count
+                FROM [books_view_all]
+                WHERE [id] = 1
+                FOR JSON PATH,
+                    INCLUDE_NULL_VALUES,
+                    WITHOUT_ARRAY_WRAPPER
+            ";
+
+            string graphQLMutationName = "deletebooks_view_all";
+            string graphQLMutation = @"
+                mutation {
+                    deletebooks_view_all(id: 1) {
+                        result
+                    }
+                }
+            ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+
+            Assert.IsTrue(actual.ToString().Contains("success"), "DwSql delete from view should be successful");
+
+            // check if entry is actually deleted
+            string dbResponse = await GetDatabaseResultAsync(msSqlQueryToVerifyDeletion);
+
+            using JsonDocument result = JsonDocument.Parse(dbResponse);
+            Assert.AreEqual(result.RootElement.GetProperty("count").GetInt64(), 0);
+        }
+
+        /// <summary>
+        /// <code>Do :</code> Delete a book from database and return the typename
+        /// <code>Check :</code>if the mutation executed successfully and returned the correct typename
+        /// </summary>
+        [TestMethod]
+        public override async Task DeleteMutationWithOnlyTypename()
+        {
+            string graphQLMutationName = "deletebook";
+            string graphQLMutation = @"
+                mutation {
+                    deletebook(id: 1) {
+                        __typename
+                    }
+                }
+            ";
+
+            string expected = @"
+              {
+                ""__typename"": ""DbOperationResult""
+              }
+            ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            SqlTestHelper.PerformTestEqualJsonStrings(expected, actual.ToString());
+        }
+        #endregion
+
+        #region Negative Tests
+
+        /// <summary>
+        /// <code>Do: </code>use an update mutation without passing any of the optional new values to update
+        /// <code>Check: </code>check that GraphQL returns an appropriate exception to the user
+        /// </summary>
+        [TestMethod]
+        public override async Task UpdateWithNoNewValues()
+        {
+            string graphQLMutationName = "updatebook";
+            string graphQLMutation = @"
+                mutation {
+                    updatebook(id: 1) {
+                        result
+                    }
+                }
+            ";
+
+            JsonElement result = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            SqlTestHelper.TestForErrorInGraphQLResponse(result.ToString(), message: $"item");
```

Comment:
can you elaborate on what this is actually validating and what error you expect to see?

Reply:
Fixed message check.